load/LoadOrganizations.py

```diff
@@ -98,101 +95,182 @@ def hasDamIds(self, languageRecord, typeCode):
 #				resultSet.add(name)
 
 
+	def get_lpts_licensors(self, language_record, fileset_id, type_code):
+		licensors = []
+		if language_record.Has_Ambiguous_Agreement():
+			print("WARN ambiguos agreement and it needs to be fixed for fileset id: %s" % (fileset_id))
+			return None
+
+		if language_record.Has_Other_Agreement():
+			print("WARN licensor: %s and co licensor: %s cannot be processed due to 'other Agreement' for fileset id: %s" % (language_record.Licensor(), language_record.CoLicensor(), fileset_id))
```

Comment:
reword
print("WARN Agreement Type is Other, which is not currently being processed.  licensor: %s and co licensor: %s,   fileset id: %s"

Reply:
@bradflood I have fixed it. See: https://github.com/faithcomesbyhearing/dbp-etl/pull/130/files#diff-0628aedffa821d1b9acf4c81d50c756fe72674fcc7ca34c47e0a5b0a65c60aceR105